mycpp/marksweep_heap.h

```diff
@@ -0,0 +1,44 @@
+#ifndef MARKSWEEP_H
+#define MARKSWEEP_H
+
+#include <new>
+#include <unordered_set>
+
+typedef uint64_t u64;
```

Comment:
Please be consistent with the codebase and just use uint64_t

Reply:
Fixed